.travis.yml

```diff
@@ -22,7 +22,8 @@ before_script:
   - cd exec
   - git clone https://github.com/fossology/fossology
   - cd fossology/src/nomos/agent/
-  - make -f Makefile.sa
+  - git checkout ba2b25ba6bc3397823ee39c35e21034fe771617d
```

Comment:
Why this line is needed ?
I made a test (https://github.com/chaoss/grimoirelab-graal/pull/21) and it seems that this line could be removed, what do you think ?